src/test/java/uk/nhs/digital/nhsconnect/lab/results/fixtures/FhirFixtures.java

```diff
@@ -0,0 +1,55 @@
+package uk.nhs.digital.nhsconnect.lab.results.fixtures;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Enumerations;
+import org.hl7.fhir.dstu3.model.Identifier;
+import org.hl7.fhir.dstu3.model.InstantType;
+import org.hl7.fhir.dstu3.model.Meta;
+import org.hl7.fhir.dstu3.model.Practitioner;
+import org.hl7.fhir.dstu3.model.UriType;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.List;
+import java.util.Optional;
+
+public final class FhirFixtures {
+
+    private static final int YEAR = 2010;
+    private static final int DAY_OF_MONTH = 24;
+    private static final int HOUR_OF_DAY = 15;
+    private static final int MINUTE = 41;
+
+    private FhirFixtures() { }
+
+    public static Optional<Practitioner> generateRequester() {
```

Comment:
These methods hide the contents for the test. E.g., when you use this method to get a requester in a test there's no reason for the reader of the test to expect that the name would be "Dr Bob Hope" until they drill down into the _Arrange_ steps. When reading a test, you should never have to look at any other code.

Could be fixed just by accepting parameters
```suggestion
    public static Optional<Practitioner> generateRequester(final String name, final AdministrativeGender gender) {
```

Then the test will
Arrange with `generateRequester("Dr Bob Hope", AdministrativeGender.MALE)`,
Act,
and Assert with `assertThat(whatever.getName()).isEqualTo("Dr Bob Hope"); assertThat(whatever.getGender()).isEqualTo(AdministrativeGender.MALE);`

And it's clear _why_ we are making those assertions.